### ISSUE TITLE
Add GeMoMa to Galaxy public server

### ISSUE THREAD

**Issue by jaredbernard** [galaxyproject/tools-iuc#5237]
Would it be possible to add GeMoMa to one of the public servers? I understand that the developers have suggested that it can be added to Galaxy, but I've checked the tools and don't see it. Am I missing something?

Thanks in advance!

**Comment by bgruening** [comment]
@jaredbernard are you referring to this too? http://www.jstacs.de/index.php/GeMoMa-Docs

The tool needs at first to be described with all its parameters, inputs and outputs into a machine-readable format. This step is still missing afaik and someone needs to find time to do that.

**Comment by jaredbernard** [comment]
Yeah, that's the one, Bjorn. Thanks so much for taking a look. I thought
they said something in their documentation about a Galaxy interface. I'm
having trouble getting it installed on my linux machine via bioconda, so I
thought it would be nice for everyone to have on Galaxy.

Or if there's another homology-based gene finder already on Galaxy, I'm
open to it.

**Comment by bgruening** [comment]
@abretaud might now of an alternative :)
